css/templates/gallery/components/_sidebar.css

```diff
@@ -14,9 +14,17 @@
  * limitations under the License.
  */
 
- .gallery-sidebar {
-   width: calc((375 / 16rem));
- }
+:root {
```

Comment:
These need to be in `page-vars.css`, or Google won't be happy

Reply:
Sure thing 👍